sample-app/src/components/DropdownSection.tsx

```diff
@@ -50,6 +76,28 @@ export default function DropdownSection({
     setFocusedOptionIndex(newIndex);
   }
 
+  function updateToPrevFocusElement() {
+    if(!decrementVerticalLinkFocus()) {
+      decrementOptionFocus();
+    }
+  }
+
+  function decrementVerticalLinkFocus(): boolean {
```

Comment:
can we document what this boolean indicates?

Reply:
It looks like it's indicating whether or not the decrement was successful. If so, can that be a separate function such as `canDecrementVerticalLinkFocus`?

Reply:
it seems a bit odd to have separate function, since a false may mean decrement/set focused link index to -1 or to the last link index of previous option but still require decrementOptionFocus. But yea I will add documentation to explain the boolean